README.md

```diff
@@ -63,6 +63,10 @@ Copy and modify the `sample_config.toml` to specify the address, username, and p
 Each `[[cameras]]` block creates a new camera; the `name` determines the RTSP path you should connect your client to.
 Currently Neolink cannot auto-detect cameras like the official clients do; you must specify their IP addresses directly.
 
+By default the h265 streaming format is used. Some cameras, for example E1, provide h264 streams, to use these you must specify `format = "h264"` in the `[[cameras]]` config.
+
+By default the HD stream is used. To swap to the SD stream add `stream = "subStream"` to the `[[cameras]]` config. You may also need to use `h264` format while using the SD stream.
```

Comment:
I'd like to have Neolink provide both the main stream and the sub stream for clients to connect to. I think this would look like `rtsp://your_server:8554/your_camera/subStream` and the corresponding `your_camera/mainStream` but I am open to ideas how this should work. Then you wouldn't need this configuration item, and you'd just connect to whatever you want in the client.

Feedback welcome, what do you think?

Reply:
That sounds reasonable but can we make `rtsp://your_server:8554/your_camera` still point to `rtsp://your_server:8554/your_camera/mainStream` as well just so that people won't need to change their scripts

Reply:
I may have this working just need to thoroughly test it